packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -37,3 +39,25 @@ export type TPasswordResetAndTradingPasswordManager = {
     platform: 'mt5' | 'dxtrade';
     account_group: 'real' | 'demo';
 };
+
+export type TResetPasswordIntent = {
+    current_list: Array<DetailsOfEachMT5Loginid> & Record<string, DetailsOfEachMT5Loginid>;
+    children({ ...props }): React.ReactElement;
```

Comment:
`React.ReactNode` would be a safer bet than `ReactElement` as it supports a greater range of elements and children.

Reply:
You're right but in this case changing ReactElement to ReactNode produces the following error. 'ResetPasswordIntent' cannot be used as a JSX component. Its return type 'ReactNode' is not a valid JSX element.